tests/director/tests/formation_constraint_api_test.go

```diff
@@ -16,6 +16,7 @@ import (
 const (
 	IsNotAssignedToAnyFormationOfTypeOperator = "IsNotAssignedToAnyFormationOfType"
 	DoesNotContainResourceOfSubtypeOperator   = "DoesNotContainResourceOfSubtype"
+	DoNotSendNotificationOperator = "DoNotSendNotification"
```

Comment:
Wouldn't it be better if we update the dependencies in the tests and use `graphql.DoNotSendNotificationOperator` instead of double-maintaining it? This applies to other operator names too.

Reply:
We don't have such value `graphql.DoNotSendNotificationOperator`. Operators are just strings for the graphQL schema. The const in the director is in the internal package. I also tried it, but it looks like we need to make the operators enum in the graphQL schema, which is quite limiting having in mind the UI constraint and the operators there which does not involve a change in compass.

Reply:
We have the strings in the `components/director/pkg/graphql/formation_constraint_validation.go` which are double-maintained to be == from the internal package.